src/ripple/protocol/STTx.h

```diff
@@ -108,6 +110,13 @@ class STTx final : public STObject, public CountedObject<STTx>
 
     Json::Value
     getJson(JsonOptions options) const override;
+
+    /**
+    If `hash` is set, will store transaction id inside the provided string.
```

Comment:
No, that's not strictly correct. Even though that's true when we do not have API version 3, it would have to be rewritten when we _do_ have it, to mean that we disable features of API versions prior to version 2, and then document similarly another enum `JsonOptions::disable_API_prior_V3` - all of which is extra work we do not really need to do _at all_, because I took care to give these enums really descriptive names. 

So, I just removed the comment instead. Thanks for spotting it!

Reply:
ok makes sense